tmt/frameworks/__init__.py

```diff
@@ -112,11 +112,14 @@ def get_pull_options(
     def extract_results(
             cls,
             invocation: 'TestInvocation',
+            results: list[tmt.result.Result],
             logger: tmt.log.Logger) -> list[tmt.result.Result]:
         """
         Extract test results.
 
         :param invocation: test invocation to which the check belongs to.
+        :param results: list of results which will be optionally saved as subresults by a
+            framework.
```

Comment:
"which will be optionally saved as subresults by a framework" - or not, who knows what framework's gonna do? Maybe they are good enough. I'd recommend sticking to what they are, not what is happening to them in the two frameworks we have today, i.e. something like "current list of results as reported by test via ``results.yaml``."

Reply:
Ok, changed to just `current list of results as reported by a test`.